<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load XLS</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>751</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneSheet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtError">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuJadda">
    <property name="title">
     <string>Jadda</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuJadda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
